ext/drsyms/drsyms_elf.c

```diff
@@ -352,7 +354,9 @@ drsym_obj_symbol_offs(void *mod_in, uint idx, size_t *offs_start OUT,
     elf_info_t *mod = (elf_info_t *) mod_in;
     if (offs_start == NULL || mod == NULL || idx >= mod->num_syms || mod->syms == NULL)
         return DRSYM_ERROR_INVALID_PARAMETER;
-    if (mod->syms[idx].st_value == 0) {
+    if ((mod->syms[idx].st_value == 0 &&
```

Comment:
Plus a comment in module_elf.c pointing back here, so if we change one we will change the other (ideally we would share the code: would have to be some API routine, unclear if that makes sense though).